aws_lambda_builders/utils.py

```diff
@@ -14,7 +14,7 @@
 LOG = logging.getLogger(__name__)
 
 
-def copytree(source, destination, ignore=None, include=None):
+def copytree(source, destination, ignore=None, include=None, maintain_symlinks=False):
```

Comment:
add the new parameter to the method docs

Reply:
missed this, thanks!

Reply:
done, and updated docstring to numpy